src/screens/Inventory/PurchaseRequest.tsx

```diff
@@ -1,92 +1,114 @@
-  const product = useSelector((state: RootState) => selectProductByInventoryId(state, purchaseRequest.inventoryId)) || EMPTY_PRODUCT;
+  const product =
+    useSelector((state: RootState) => selectProductByInventoryId(state, purchaseRequest.inventoryId)) || EMPTY_PRODUCT;
   const userIsAdmin = useSelector(selectCurrentUserIsAdmin);
   const currentUserId = useSelector(selectCurrentUserId);
 
   const requester = useSelector((state: RootState) => selectSiteUserById(state, purchaseRequest.requesterId));
 
-
   // If no purchase request was passed in (i.e. reaching this URL directly), redirect to InventoryMain
   if (!props.location.state?.purchaseRequest || !product) {
     return <Redirect to={'/inventory'} />;
   }
 
   const handleSubmit = (purchaseRequest: PurchaseRequestRecord, approved: boolean) => {
-    const reviewData = {
-      reviewerId: currentUserId,
-      reviewedAt: moment().toISOString(),
-      status: approved ? PurchaseRequestStatus.APPROVED : PurchaseRequestStatus.DENIED,
-    };
-    updatePurchaseRequest(purchaseRequest.id, reviewData);
-    updatePurchaseRequestInRedux({ id: purchaseRequest.id, ...reviewData });
+    reviewPurchaseRequest(purchaseRequest, approved, currentUserId);
     history.goBack();
   };
 
   return (
     <BaseScreen title="Inventory Receipt" leftIcon="backNav">
       <BaseScrollView>
-        <div className={classes.content}>
-          <Typography variant="h3">{`Purchase Request for ${product.name}`}</Typography>
-          <Typography variant="body1">{`Request status: ${purchaseRequest.status}`}</Typography>
-          <Typography variant="body1">{`Amount purchased ${purchaseRequest.amountPurchased} ${product.unit}(s)`}</Typography>
-          <Typography variant="body1">{`Amount spent ${purchaseRequest.amountSpent} ks`}</Typography>
-          <Typography variant="body1">{`Created at ${formatDateStringToLocal(purchaseRequest.createdAt)}`}</Typography>
-          <Typography variant="body1">{`Submitted by ${requester?.username || purchaseRequest.requesterId}`}</Typography>
-          {purchaseRequest.notes && <Typography variant="body1">{`Notes: ${purchaseRequest.notes}`}</Typography>}
-          {purchaseRequest.receipt && <img className={classes.imageContainer} src={purchaseRequest.receipt[0].url} alt="receipt"/> }
-          {userIsAdmin && purchaseRequest.status == PurchaseRequestStatus.PENDING && (
-            <div>
-              <Button onClick={() => handleSubmit(purchaseRequest, true)} label={'Approve'} />
-              <Button onClick={() => handleSubmit(purchaseRequest, false)} label={'Deny'} />
-            </div>
-          )}
+        <div className={classes.headerContainer}>
+          <InventoryInfo productId={product.id} lastUpdated={getInventoryLastUpdated(purchaseRequest.inventoryId)} />
+          <div className={classes.reviewButtonsContainer}>
+            {userIsAdmin && purchaseRequest.status == PurchaseRequestStatus.PENDING
```

Comment:
NIT: is there a reason it's double equals here? I was under impression that best practice was triple equals always. We have some instances of double equals in the codebase but it's probably a low priority cleanup